cmd/atlas/internal/migratelint/run.go

```diff
@@ -226,7 +225,12 @@ var (
 			}
 			return append(lines, strings.Join(words[j:], " "))
 		},
-	})
+		"cyan":         color.CyanString,
+		"green":        color.HiGreenString,
+		"red":          color.HiRedString,
+		"redBgWhiteFg": color.New(color.FgHiWhite, color.BgHiRed).SprintFunc(),
+		"yellow":       color.YellowString,
+	}
```

Comment:
Why? This isn't DRY anymore.

Reply:
I'm moving all serialized logged objects to cmdlog, and new parts depend on `migratelint` report objects, which creates me circular dep.

Reply:
Eventually, the `migratelint` templates will be moved to this as well.